### PR TITLE
Add null check in  FKUserDefaultsSwizzleUtility.m

### PR DESCRIPTION
**Pull request by MichelBahl**
Prevent Flipper from runtime exception by add null c.heck

### CODE REVIEW

**Comment by passy** [comment]
@MichelBahl Would you mind signing the CLA so we can import this please? :)

**Comment by MichelBahl** [comment]
> @MichelBahl Would you mind signing the CLA so we can import this please? :)

Done

File: iOS/Plugins/FlipperKitUserDefaultsPlugin/FKUserDefaultsSwizzleUtility.m [hunk 1]
```diff
@@ -59,7 +59,9 @@ - (void)swizzleSelector:(SEL)selector
         NSString* originalStr =
             [@"comfacebookFlipperKit_" stringByAppendingString:selStr];
         [invocation setSelector:NSSelectorFromString(originalStr)];
-        block(invocation);
```
Comment (priteshrnandgaonkar) [inline iOS/Plugins/FlipperKitUserDefaultsPlugin/FKUserDefaultsSwizzleUtility.m:62]: shouldn't this be just `blk(invocation)`

**Review by priteshrnandgaonkar — COMMENTED** [review]
I am not sure if this is the right fix.

**Pull request closed without merging**